aten/src/ATen/native/native_functions.yaml

```diff
@@ -6592,6 +6592,13 @@
     MkldnnCPU: mkldnn_reorder_conv3d_weight
   autogen: mkldnn_reorder_conv3d_weight.out
 
+- func: reorder_linear_weight(Tensor self, int? batch_size) -> Tensor
+  variants: function
+  python_module: nn
+  dispatch:
+    MkldnnCPU: reorder_linear_weight
+  autogen: reorder_linear_weight.out
+
```

Comment:
Can we register it inside MKLDNNConversion.cpp without changing native_functions.yaml? I would suggest we make it a private interface first since it is only used inside TorchInductor right now, not user visible.

Reply:
Yes, removed it.